bin.src/om10_lensing_equations.py

```diff
@@ -0,0 +1,168 @@
+import os
+import numpy as np
+import pylab as pl
+from scipy.interpolate import interp1d
+#--------------------------------------------------------------------
+from astropy.cosmology import WMAP7 as p15
+
```

Comment:
Please include an `__all__` declaration in this module defining which methods should be imported by
```
from om10_lensing_equations import *
```